corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -97,7 +105,7 @@ def process_change(self, change):
                 return
 
             if doc.get('doc_type') is not None and doc['doc_type'].endswith("-Deleted"):
-                self._delete_doc_if_exists(change.id)
+                self._delete_doc_if_exists(change.id, domain=doc.get('domain'))
```

Comment:
why does this not need to check the change meta like the above did?

Reply:
I think it is because this isn't a deletion, and therefore there should still be a valid DB object when we call `change.get_document()`.